v19.2/drop-table.md

```diff
@@ -116,15 +149,41 @@ DROP TABLE
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> SHOW TABLES FROM bank;
+> SHOW TABLES FROM movr;
 ~~~
 
 ~~~
-+------------+
-| table_name |
-+------------+
-+------------+
-(0 rows)
+          table_name
++----------------------------+
+  rides
+  user_promo_codes
+  vehicle_location_histories
+  vehicles
+(4 rows)
+~~~
+
```

Comment:
It might be good to add some explication here between these statements like "Now, we can verify the FK has been removed by running SHOW CREATE again and seeing that the XXX_fk is gone" ... or similar.

Reply:
Done!